src/coreComponents/constitutive/fluid/multifluid/PVTDriver.hpp

```diff
@@ -99,7 +100,8 @@ class PVTDriver : public TaskBase
 
   enum columnKeys { TIME, PRES, TEMP }; ///< Enumeration of "input" column keys for readability
 
-  static constexpr real64 m_baselineTol = 1e-3; ///< Comparison tolerance for baseline results
+  static constexpr real64 m_baselineTol = constitutive::MultiFluidConstants::baselineTolerance; ///< Comparison tolerance for baseline
```

Comment:
not sure this shortcut needs to be kept. not sure if it is necessary to have the same litteral value stored in the binary 2 times.